File: single_number.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        int count[30000]={0};
        for(int i=0;i<nums.size();i++){
            count[nums[i]]++;
        }
        for(int i=0;i<30000;i++){
            if(count[i]==1)
                return i;
        }
        return 0;
    }
};

File: singleNumber.java
class Solution {
    public int singleNumber(int[] nums) {
        Map<Integer,Integer> mp=new HashMap<>();
        for(int i=0;i<nums.length;i++){
            if(mp.containsKey(nums[i])){
                mp.put(nums[i],mp.get(nums[i])+1);
            }
            else{
                mp.put(nums[i],1);
            }
        }
        
        return -1;
    }
}